docker/1.14/Dockerfile.eia

```diff
@@ -0,0 +1,86 @@
+FROM ubuntu:16.04
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+ARG TFS_SHORT_VERSION=1.14
+ARG TFS_VERSION=1-14-0
+ARG TFS_EI_VERSION=1-2
+ARG PYTHON=python3
+ARG PIP=pip3
+ARG PYTHON_VERSION=3.6.6
+ARG HEALTH_CHECK_VERSION=1.3.3
+
+# nginx + njs
+RUN apt-get update && \
+    apt-get -y install --no-install-recommends \
+    curl \
+    gnupg2 \
+    ca-certificates \
+    git \
+    wget \
+    vim \
+    build-essential \
+    zlib1g-dev && \
+    curl -s http://nginx.org/keys/nginx_signing.key | apt-key add - && \
+    echo 'deb http://nginx.org/packages/ubuntu/ xenial nginx' >> /etc/apt/sources.list && \
+    apt-get update && \
+    apt-get -y install --no-install-recommends nginx wget nginx-module-njs  && \
+    apt-get clean 
+
+# See http://bugs.python.org/issue19846
+ENV LANG C.UTF-8
+
+RUN wget https://www.python.org/ftp/python/$PYTHON_VERSION/Python-$PYTHON_VERSION.tgz && \
+    tar -xvf Python-$PYTHON_VERSION.tgz && cd Python-$PYTHON_VERSION && \
+    ./configure && make && make install && \
+    apt-get update && apt-get install -y --no-install-recommends libreadline-gplv2-dev libncursesw5-dev libssl-dev libsqlite3-dev tk-dev libgdbm-dev libc6-dev libbz2-dev && \
+    make && make install && rm -rf ../Python-$PYTHON_VERSION* && \
+    ln -s /usr/local/bin/pip3 /usr/bin/pip
```

Comment:
Same here, let's cleanup the layers and add one instruction per line to improve readability.

Reply:
Done. Thanks.